README.md

```diff
@@ -24,7 +24,9 @@ This tap:
 
 1. Install
 
-    pip install tap-shopify
+   ```sh
```

Comment:
Nice catch with the formatting here!  We are working on changes to the "Install" step of this doc though, so if you could revert this change as to avoid potential merge conflicts later.